@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf

```diff
@@ -19,32 +19,34 @@ Author:
 	BradMick
 ---------------------------------------------------------------------------- */
 params ["_heli", "_deltaTime", "_altitude", "_temperature", "_dryAirDensity"];
-
-private _rtrPos              = [0.0, 2.06, 0.70];
-private _rtrHeightAGL        = 3.606;   //m
-private _rtrDesignRPM        = 289.0;
-private _rtrRPMTrimVal       = 1.01;
-private _rtrGearRatio        = 72.29;
-private _rtrNumBlades        = 4;
-
-private _bladeRadius         = 7.315;   //m
-private _bladeChord          = 0.533;   //m
-private _bladePitch_min      = 1.0;     //deg
-private _bladePitch_max      = 19.0;    //deg
-
-private _rtrPowerScalarTable = [
-                                [   0, 1.777]
-                               ,[2000, 1.617]
-                               ,[4000, 1.530]
-                               ,[6000, 1.377]
-                               ,[8000, 1.284]
-                               ];
-private _rtrGndEffModifier   = 0.238;
-private _rtrThrustScalar_min = 0.120;
-private _rtrThrustScalar_max = 1.830;   //20,200lbs @ 6700ft, 15 deg C and 0.9 collective
-
-private _altitude_max        = 30000;   //ft
-private _baseThrust          = 102302;  //N - max gross weight (kg) * gravity (9.806 m/s)
+#include "\fza_ah64_sfmplus\headers\core.hpp"
+
+private _rtrPos                 = [0.0, 2.06, 0.70];
+private _rtrHeightAGL           = 3.606;   //m
+private _rtrDesignRPM           = 289.0;
+private _rtrRPMTrimVal          = 1.01;
+private _rtrGearRatio           = 72.29;
+private _rtrNumBlades           = 4;
+
+private _bladeRadius            = 7.315;   //m
+private _bladeChord             = 0.533;   //m
+private _bladePitch_min         = 1.0;     //deg
+private _bladePitch_max         = 19.0;    //deg
+
+private _rtrPowerScalarTable    = [
+                                   [   0, 1.777]
+                                  ,[2000, 1.617]
+                                  ,[4000, 1.530]
+                                  ,[6000, 1.377]
+                                  ,[8000, 1.284]
+                                  ];
+private _rtrGndEffModifier      = 0.238;
+private _rtrThrustScalar_min    = 0.120;
+private _rtrThrustScalar_max    = 1.830;   //20,200lbs @ 6700ft, 15 deg C and 0.9 collective
+private _rtrAirspeedVelocityMod = 0.4;
+
+private _altitude_max           = 30000;   //ft
+private _baseThrust             = 102302;  //N - max gross weight (kg) * gravity (9.806 m/s)
```

Comment:
One of the reason why I don't like this alignment technique because changing one line requires changing all which makes the diffs massive.